src/HackerNews/MainPage/MainPage.tsx

```diff
@@ -0,0 +1,192 @@
+import React, { useEffect, useState } from "react";
+import NewsSummary from "./NewsSummary/NewsSummary";
+import NewsInfo from "./NewsInfo/NewsInfo";
+
+import { Route, Routes } from "react-router-dom";
+
+import styled from "styled-components";
+
+export interface response {
```

Comment:
`interface` и `type` в `PascalCase`
Ну и возможно название можно менее абстрактней сделать)
```suggestion
export interface NewsItem {
```